packages/ingest/aws.js

```diff
@@ -99,7 +99,7 @@ function sqs(local) {
 
 class Events {
   static async putEvent(name, schedule, state, description = null, role = null) {
-    const cwevents = new aws.cloudwatchevents();
+    const cwevents = new AWS.CloudWatchEvents();
```

Comment:
I think the typo is actually with `new` - e.g. the intention was:
```javascript
const cwevents = aws.cloudwatchevents();
```
I think I made this updates for tests

Reply:
Should that change then not also apply to the `deleteEvent`, `putTarget` and `deleteTarget` functions?

Reply:
I think the revert is okay for now - I think whatever tests I was writing ended up not getting rolled into the test suite (as CI seems to be passing with this change) - thanks for checking though!